tket/src/Utils/UnitID.cpp

```diff
@@ -76,6 +83,12 @@ const std::string& c_default_reg() {
   return *regname;
 }
 
+const std::string& w_default_reg() {
+  static std::unique_ptr<const std::string> regname =
+      std::make_unique<const std::string>("_wasm_wire");
```

Comment:
I wanted to make sure that this not named in a way that it could have the identical name with some other register. If You are not concerned about that I am happy to rename it.

Reply:
I assumed it would be fine if it clashed with the name of another `Qubit` or `Bit` register, but it seems I was wrong -- register names must be globally unique. Perhaps the underscore is enough (`_w`)?

Reply:
(If there is someone complaining that this name is used to much for other things we can do a rename in the future.)